openarc/openarc.c

```diff
@@ -3646,13 +3650,33 @@ mlfi_eom(SMFICTX *ctx)
 		/*
  		**  Authentication-Results
 		*/
+		u_char arcchainbuf[ARC_MAXHEADER + 1];
```

Comment:
Define at top of function.  I realize this works; it's just legacy coding style.